### ISSUE TITLE
Repo Status?

### ISSUE THREAD

**Issue by 5nyper** [tomauty/react-native-chart#160]
Is this Repo still being actively maintained? If so, when will the addition of `charTitle` be available?

**Comment by vccabral** [comment]
This has been around 6 months. @Vikaton did you find an alternative?

**Comment by 5nyper** [comment]
No, I ended up just using this and altering the library just a tiny bit @vccabral

**Comment by tomauty** [comment]
As a generic FYI — I am no longer able to maintain this library. I recommend checking out [victory-native](https://github.com/FormidableLabs/victory-native) as it's much more maintained.
